modules/rosa-sts-account-wide-roles-and-policies.adoc

```diff
@@ -658,6 +658,87 @@ The account number present in the `sts_installer_trust_policy.json` and `sts_sup
 ----
 ====
 
+.ROSA `ocm-role` and policy file
+[cols="1,2",options="header"]
+|===
+
+|Resource|Description
+
+|`ManagedOpenShift-OCM-Role`
+|An IAM role that enables you to create and maintain ROSA clusters in  {cluster-manager}.
```

Comment:
"Enables you" is a form of anthrophomorphism. How about:

> With this IAM role enabled, you can create and maintain ROSA clusters in {cluster-manager}.

![Screenshot from 2024-07-01 12-33-44](https://github.com/openshift/openshift-docs/assets/57954076/05e79eab-761a-448d-a33c-27182397554d)

Reply:
Should a "ROSA" attribute be used here?

![Uploading Screenshot from 2024-07-01 12-39-07.png…]()